lms/djangoapps/projects/management/commands/tests/test_migrate_project_courseids_v2.py

```diff
@@ -5,19 +5,21 @@
 from datetime import datetime
 import uuid
 
+from django.conf import settings
 from django.contrib.auth.models import Group, User
 from django.test import TestCase
```

Comment:
Do you still need TestCase here?  Be sure to execute both 'paver run_quality' and 'paver run_pep8' in your devstack.